src/App.tsx

```diff
@@ -3,28 +3,16 @@ import { reatomContext } from '@reatom/react';
 import { StrictMode } from 'react';
 import { store } from '~core/store/store';
 import { AuthWrapper } from '~core/auth';
-import { runGoogleTagManager } from '~utils/metrics/tagManager';
 import { Router } from '~core/router';
-
+import { CookieConsentBanner } from '~features/cookie_consent_banner';
 const root = createRoot(document.getElementById('root')!);
 root.render(
   <StrictMode>
     <reatomContext.Provider value={store}>
       <AuthWrapper>
         <Router />
+        <CookieConsentBanner />
```

Comment:
hm, can't we move it to other common features place in src/views/CommonView.tsx ? 🤔 
not sure it's critical, but I would expect it there

Reply:
fixed